### PR TITLE
Add new pkg LSoftTechnologies.ActiveDataBurner version 6

### PR DESCRIPTION
**Pull request by richardsondev**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16512)

### CODE REVIEW

File: manifests/l/LSoftTechnologiesInc/ActiveDataBurner/6/LSoftTechnologiesInc.ActiveDataBurner.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: LSoftTechnologiesInc.ActiveDataBurner
```
Comment (ItzLevvie) [inline manifests/l/LSoftTechnologiesInc/ActiveDataBurner/6/LSoftTechnologiesInc.ActiveDataBurner.yaml:3]:
```suggestion
PackageIdentifier: LSoftTechnologies.ActiveDataBurner
```

File: manifests/l/LSoftTechnologiesInc/ActiveDataBurner/6/LSoftTechnologiesInc.ActiveDataBurner.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate
+
+PackageIdentifier: LSoftTechnologiesInc.ActiveDataBurner
```
Comment (ItzLevvie) [inline manifests/l/LSoftTechnologiesInc/ActiveDataBurner/6/LSoftTechnologiesInc.ActiveDataBurner.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: LSoftTechnologies.ActiveDataBurner
```

File: manifests/l/LSoftTechnologiesInc/ActiveDataBurner/6/LSoftTechnologiesInc.ActiveDataBurner.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: LSoftTechnologiesInc.ActiveDataBurner
```
Comment (ItzLevvie) [inline manifests/l/LSoftTechnologiesInc/ActiveDataBurner/6/LSoftTechnologiesInc.ActiveDataBurner.installer.yaml:3]:
```suggestion
PackageIdentifier: LSoftTechnologies.ActiveDataBurner
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

**Review by ghost — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**